udata/core/dataset/models.py

```diff
@@ -667,6 +668,15 @@ def update_resource(self, resource):
         self.reload()
         post_save.send(self.__class__, document=self)
 
+    def remove_resource(self, resource):
+        # Deletes resource's file from file storage
+        if resource.fs_filename is not None:
+            storages.resources.delete(resource.fs_filename)
+
+        self.resources.remove(resource)
+        self.last_modified = datetime.now()
```

Comment:
Keep `last_modified` out of this fn, like here https://github.com/opendatateam/udata/blob/master/udata/core/dataset/api.py#L267

When using the CLI we usually do not want to bump `last_modified`.

I'm not sure if we should keep the `save` here. It would probably be more foolproof. Maybe look at other examples.

Reply:
👍

Reply:
Regarding the tests the behavior and the storage is already tested in api tests. Do we want a specific model level test?

Reply:
Maybe test the function directly, just to make sure.